### DISCUSSION TITLE
Center the x-axis labels to the columns in a bar diagram

### DISCUSSION THREAD

**Discussion by droidgren** [dbuezas/lovelace-plotly-graph-card#344] [Q&A]
Is it possible to center the x-axis labels to the columns in a bar diagram?
Also is possible to change the x-axis labels?
![image](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/354317/1d1bf8ec-a5b5-46fa-a87a-90345c1ab1f4)

**Comment by dbuezas** [comment]
Yes see the axes docs https://plotly.com/javascript/reference/layout/xaxis/#layout-xaxis-tickangle
